templates/taxi/driver_detail.html

```diff
@@ -1,23 +1,24 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>
-    Username: {{ driver.username }}
-  </h1>
+  <h1>Username: {{ driver.username }}</h1>
 
   <p><strong>First name:</strong> {{ driver.first_name }}</p>
   <p><strong>Last name:</strong> {{ driver.last_name }}</p>
   <p><strong>License number:</strong> {{ driver.license_number }}</p>
   <p><strong>Is staff:</strong> {{ driver.is_staff }}</p>
 
+  <a style="float:right" href="{% url 'taxi:driver-update' pk=driver.id %}">Update license number</a>
+  <a style="float:right" href="{% url 'taxi:driver-delete' pk=driver.id %}">Delete driver</a>
+
   <div class="ml-3">
     <h4>Cars</h4>
 
     {% for car in driver.cars.all %}
```

Comment:
The expression `driver.cars.all` should be `driver.cars.all()` to properly call the method and retrieve the queryset of cars associated with the driver.